nginx.conf

```diff
@@ -37,7 +36,7 @@ http {
       return 301 $scheme://$host/remote.php/dav;
     }
 
-    client_max_body_size 1G;
+    client_max_body_size 10G;
```

Comment:
hum, this one actually, I make people pay more for this!
I don't want anybody upload a file of 4G to my servers, except if I'm aware, and prepared!